orc/graphs/general.py

```diff
@@ -88,100 +90,116 @@ def completeness_checker(state: GeneralModState) -> dict:
 
     def completeness_decision(state: GeneralModState) -> str:
         """
-        Decide whether the answer is already good enough or needs additional context from Bing search.
+        Decide whether the answer is already good enough or needs additional context from google search.
 
         Args:
             state (GeneralModState): The current state of the general model.
 
         Returns:
-            str: The next action to take, either "bing_web_search" or "__end__".
+            str: The next action to take, either "google_web_search" or "__end__".
         """
         if state["relevance"] == "no":
             if verbose:
                 print("---DECISION: CONDUCT WEB SEARCH TO ADD CONTEXT TO ANSWER---")
-            return "bing_web_search"
+            return "google_web_search"
         else:
             if verbose:
                 print("---DECISION: GENERATED ANSWER IS GOOD ENOUGH---")
             return "__end__"
+        
+    
+    google_search = GoogleSerperAPIWrapper(k=3, serper_api_key= GOOGLE_SEARCH_API_KEY)
```

Comment:
this is nice. you could also consider adding a comment that mentions what this line is doing, for example:

`# setting up the api wrapper to interact with the google search api where k is the number of search results to be returned`